src/components/admin/form-components/ListSelector.jsx

```diff
@@ -8,26 +8,22 @@ import MenuItem from 'material-ui/MenuItem';
 export default class ListSelector extends React.Component {
   constructor(props) {
     super(props);
-    this.onChange = this.onChange.bind(this);
-  }
-
-  onChange(event, ...restArgs) {
-    this.props.update(restArgs[1]);
+    this.onChange = (event, index, value) => this.props.update(value);
   }
 
   render() {
     return (
       <SelectField
         floatingLabelText={this.props.type}
         maxHeight={400}
-        value={this.props.chosenType}
+        value={this.props.chosenElement || 'none'}
```

Comment:
Yeah so this is what I was afraid of when I was saying we should think about how to do this. Now this component is dependent on the fact that `elements` contains a `none` element right?

Reply:
Let me do a quick spike to check on this. I don't actually believe so, but will confirm and get back to you.